student/src/main/java/tds/student/services/remote/RemoteOpportunityService.java

```diff
@@ -173,7 +214,23 @@ public ApprovalInfo checkSegmentApproval(final OpportunityInstance oppInstance)
 
   @Override
   public void denyApproval(final OpportunityInstance oppInstance) throws ReturnStatusException {
-    legacyOpportunityService.denyApproval(oppInstance);
+  
+    if(isLegacyCallsEnabled) {
+      legacyOpportunityService.denyApproval(oppInstance);
+    }
+
+    if (!isRemoteExamCallsEnabled) {
+      return;
+    }
+    
+    OpportunityStatus opportunityStatus = getStatus(oppInstance);
+  
+    /* Conditional on line [257] */
```

Comment:
Can you add the class name?  I understand it is in the legacy opportunity service but the comment will make more sense for people that aren't in the know.